<launch>
    <node pkg="pvdf" type="pvdf_scan" name="pvdf_scan" output="screen"/>
    <node pkg="rqt_plot" type="rqt_plot" name="show_pvdf_msg" output="screen"/>
</launch>
